examples/chart/teleport-kube-agent/values.yaml

```diff
@@ -65,6 +65,16 @@ teleportVersionOverride: ""
 # certificate.
 insecureSkipProxyTLSVerify: false
 
+# Settings for mounting your own TLS material in the agent pod.
+# The agent does not expose a TLS server, so this is only used to trust CAs.
+tls:
+  # Name of an existing secret to use which contains a CA or trust bundle in x509 PEM format.
+  # This is useful to trust private CAs.
+  # This will automatically set the SSL_CERT_FILE environment variable to trust the CA.
+  # Create the secret with `kubectl create secret generic --from-file=ca.pem=/path/to/root-ca.pem`
+  # The filename inside the secret is important - it _must_ be ca.pem
+  existingCASecretName: ""
```

Comment:
These CAs are primarily for database agent configuration and since an agent can proxy multiple databases, each potentially with its own CA, should we make this a list?

Reply:
Supporting multiple secrets means not being consistant between `teleport-cluster` and `teleport-kube-agent`.

The two ways of trusting multiple CAs are using `SSL_CERT_DIR` or building a CA bundle. The latter would require an active component (or an init container, but this would handle poorly cert renewal & content change). I think only `SSL_CERT_DIR` is viable here. `SSL_CERT_FILE` does not support comma-separated list.

However users don't need this feature for database trust, this is already supported by the chart:
- when declaring their DB in `databases` they can specify a CACert location
- they can already mount their CA certs using `extraVolumes`
I think this might be better for database users to rely only on `database.tls.ca_cert` instead of a global trust store as certs passed in `SSL_CERT_*` as those certs are also used to trust the Teleport Proxy. The only limitation with this approach is that is does not generalize to Application Access.

I would tend to document properly in `database` an example of private CA trust for a specific database.